frontend/website/pages/docs/faq.mdx

```diff
@@ -81,6 +81,35 @@ No, Coda does not natively implement privacy features at the moment. However, pr
 
 [See here](/docs/node-operator#running-coda-as-a-service) for information about using launchd or systemd with coda.
 
+## Block Producers
+
+#### Can I run multiple Block Producers with the same Private Key? 
+
+No. It is important to only run one Block Producer process against a particular private key. In the event the Coda Daemon receives two different blocks from the same Block Producer, both blocks would be discarded. 
```

Comment:
```suggestion
No. It is important to only run one Block Producer process against a particular private key. In the event the Coda daemon receives two different blocks from the same Block Producer, both blocks would be discarded. 
```

Tiny nit to be consistent with pages such as https://codaprotocol.com/docs/node-operator#staking-coda